app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background"
+    tools:context=".MainActivity">
+
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar2"
+        style="@style/MainToolbar"
+        android:layout_width="match_parent"
```

Comment:
[Important: MATCH_PARENT is not recommended for widgets contained in a ConstraintLayout](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout)